src/components/NavBar/SearchBar.tsx

```diff
@@ -33,6 +34,9 @@ export const SearchBar = () => {
   const phase1Flag = useNftFlag()
   const isMobile = useIsMobile()
   const isTablet = useIsTablet()
+  const isPhase1 = phase1Flag === NftVariant.Enabled
+  // when removing this flag, copy content of SearchBarNft.css into SearchBar.css and use the styles import only
+  const finalStyles = isPhase1 ? nftStyles : styles
```

Comment:
a lot cleaner approach would be to create individual styles for the differences such as the borderWidth and the breakpoint widths and use the phase1 conditional to switch between them instead of switching the entire file and duplicating a lot of code